pkg/debug/scripts/collect-info.sh

```diff
@@ -82,7 +88,15 @@ sort_cat_jq()
     # Also:
     # "-R (. as $line | try fromjson)" means ignore a line if is not a JSON,
     # "(nanos // 0)" means return 0 if nanos is null
-    jq -R '(. as $line | try fromjson) | .timestamp.str.nanos = ((.timestamp.nanos // 0) + 1e9 | tostring | .[1:]) | .timestamp.str.human = (.timestamp.seconds | strftime("%B %d %Y %I:%M:%S")) | .timestamp.str = "\(.timestamp.str.human).\(.timestamp.str.nanos)"'
+    if [ -n "$OUT_LOGS_IN_JSON" ]; then
+        jq -R '(. as $line | try fromjson) | .timestamp.str.nanos = ((.timestamp.nanos // 0) + 1e9 | tostring | .[1:]) | .timestamp.str.human = (.timestamp.seconds | strftime("%B %d %Y %I:%M:%S")) | .timestamp.str = "\(.timestamp.str.human).\(.timestamp.str.nanos)"'
+    else
+        # "(.content as $cont | $cont | try (fromjson.msg) catch $cont)"
+        #     - we handle nested JSON, which is not always the case
+        # | awk /./
+        #     - remove blank lines, kernel log has additional \n at the end
+        jq -r -R '(. as $line | try fromjson) | .timestamp.str.nanos = ((.timestamp.nanos // 0) + 1e9 | tostring | .[1:4]) | .timestamp.str.human = (.timestamp.seconds | strftime("%b-%d-%y %I:%M:%S")) | "\(.timestamp.str.human).\(.timestamp.str.nanos)|\(.severity)|\(.source)|\(.filename // "")| \(  (.content as $cont | $cont | try (fromjson.msg) catch $cont) )"' | awk /./
```

Comment:
I'm not even going to try to review this :)

Reply:
Forget about this. I can't answer any questions anyway.